frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -0,0 +1,38 @@
+# This file consists of guide lines to create automation scripts
+
+## Scenario files
+
+1. Only validations should be present
+2. Don't use test data directly - It needs to be passed via data files (will use .ts files for now )
+
+## View files
+
+1. Page objects should be id, css selectors, buttontext etc.. [No XPath]- Already following
+    * Each section should have one object as shown below (It helps to reduce the import list in scenarios)
+
+      export const deleteDeployPopupObj = {
+          form: element(by.css('form.modal-content')),
+          checkbox: element(by.css('input[type="checkbox"]')),
+          cancel: element(by.css('[data-test-id="modal-cancel-action"]')),
+          delete: element(by.css('#confirm-action'))
+      }
```

Comment:
We actually want to avoid class name selectors. Maybe make a note about that. We cannot avoid using them at this time until we add more test IDs.
ID selectors will be few as we want to avoid having them in components.

Reply:
Sure christian, Once test id's are added we will update the scripts